app/assets/stylesheets/arclight/modules/search_results.scss

```diff
@@ -150,6 +150,24 @@ article.document {
   color: $body-color;
 }
 
+.al-search-result-index-article .breadcrumb-links {
+  .col {
+  flex-basis: auto;
+  margin-left: -12px;
```

Comment:
I was directed to refer to the classes in https://getbootstrap.com/docs/4.3/utilities/spacing/#notation for some rules I've applied involving `margin-left`, but those may have been specific to my cases.

Reply:
Thanks @jrgriffiniii, I'll try those classes.

Reply:
@jrgriffiniii, used those spacing classes allowed me to clean up my scss. 👍